### PR TITLE
feat: add nuxt-unleash

### PR DESCRIPTION
**Pull request by conejerock**
Add module to use Unleash toggle feature services

Repo: https://github.com/conejerock/nuxt-unleash

### CODE REVIEW

**Comment by Atinux** [comment]
Thanks @conejerock 

Any plan for making it Nuxt 3 compatible?

**Comment by conejerock** [comment]
> 

I still don't know enough about Nuxt3 to integrate it, I'll try it soon

File: modules/nuxt-unleash.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: nuxt-unleash
+description: Nuxt.js module to use Unleash toggle feature services
+repo: conejerock/nuxt-unleash
+npm: nuxt-unleash
+icon: nuxt-unleash.svg
+github: https://github.com/conejerock/nuxt-unleash
+website: https://github.com/conejerock/nuxt-unleash
+learn_more: ''
```
Comment (pi0) [inline modules/nuxt-unleash.yml:8]:
```suggestion
learn_more: 'https://github.com/Unleash/unleash'
```

**Comment by pi0** [comment]
Thanks for submitting your nice module @conejerock. I've created https://github.com/conejerock/nuxt-unleash/issues/3 to track Nuxt 3 support also added link to unleash library (+ https://github.com/conejerock/nuxt-unleash/pull/4) please let me know if it is wrong url.

**Pull request merged**